vllm/config.py

```diff
@@ -1208,7 +1212,7 @@ def create_draft_parallel_config(
         elif speculative_draft_tensor_parallel_size != 1:
             # TODO(wooyeon): allow tp values larger than 1
             raise ValueError(
-                f"{speculative_draft_tensor_parallel_size=} cannot be"
+                f"{speculative_draft_tensor_parallel_size=} cannot be "
```

Comment:
nit

Reply:
This is a typo I found when I was working on(#6300). We need this space or we will print 'cannot beother value than 1'. The right one is supposed to be 'cannot be other value than 1'.